locale/en/site.json

```diff
@@ -39,6 +39,10 @@
             "link": "about/governance",
             "text": "Governance"
```

Comment:
Note: The TSC is still under a tab called "Governance". We are under a new side bar called "Community". There is definitely a better IA for the site. Figured we'd save that for a larger re-design though. Can anyone think of a better stop-gap solution?

Reply:
I really can't, but totally open to suggestions 😬